Aardvark.xcodeproj/project.pbxproj

```diff
@@ -212,6 +248,7 @@
 			children = (
 				EAD1442419E073FB0065A1FF /* Aardvark.h */,
 				EAD1442619E073FB0065A1FF /* Aardvark.m */,
+				4551A2661BD9B93B00F216D0 /* ARKLogType.h */,
```

Comment:
oh I see. you created a new file that houses the logging types. Not sure how I feel about that yet.